link_shortener/templates/all_links.html

```diff
@@ -9,6 +9,12 @@
       </a>
     </div>
     <br><br>
+
+      <!-- How to retrieve owners:
+  {% for owner in data["owners"] %}
+      {{ owner }}
+  {% endfor %}-->
```

Comment:
Toto je pořád punk-rock, ale mergnu to i s tím možná 😄

Reply:
3e0cf5aa84e9138503811547f7b985aaef212a84